src/sentry/api/validators/external_actor.py

```diff
@@ -31,9 +31,7 @@ def validate_external_id_option(external_id: Optional[str]) -> Optional[str]:
 
 def validate_integration_id_option(
     integration_id: Optional[str], organization: Organization
```

Comment:
You should create a new function `validate_integration_id` that does everything `validate_integration_id_option` does and then you can just have something like 
```
def validate_integration_id_option(...):
    if not integration_id:
        return None
	return validate_integration_id(...)
```

Reply:
Alternatively you could just rename `validate_integration_id_option` -> `validate_integration_id` in this case.